client/src/test/resources/config/client.yml

```diff
@@ -14,34 +17,72 @@ tls:
   loadTrustStore: true
   # trust store location can be specified here or system properties javax.net.ssl.trustStore and password javax.net.ssl.trustStorePassword
   trustStore: client.truststore
+  # trust store password
+  trustStorePass: password
   # key store contains client key and it should be loaded if two-way ssl is uesed.
   loadKeyStore: true
   # key store location
   keyStore: client.keystore
+  # key store password
+  keyStorePass: password
+  # private key password
+  keyPass: password
+# settings for OAuth2 server communication
 oauth:
+  # OAuth 2.0 token endpoint configuration
   token:
-    tokenRenewBeforeExpired: 4000
-    expiredRefreshRetryDelay: 5000
-    earlyRefreshRetryDelay: 30000
-    # static token service url in case the service is deployed to a VM. It will be used if it is configured here.
-    server_url: http://localhost:7777
-    # serviceId for the token service, it will be used to do the lookup if above server_url is not configured.
+    cache:
+      #capacity of caching TOKENs
+      capacity: 200
+    # The scope token will be renewed automatically 1 minutes before expiry
+    tokenRenewBeforeExpired: 60000
+    # if scope token is expired, we need short delay so that we can retry faster.
+    expiredRefreshRetryDelay: 2000
+    # if scope token is not expired but in renew windown, we need slow retry delay.
+    earlyRefreshRetryDelay: 4000
+    # token server url. The default port number for token service is 6882.
+    server_url: https://localhost:6882
```

Comment:
changing port may cause test fail

Reply:
I just blindly copied and it was my mistake. Thanks for pointing it out.